GestureHandler.web.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {
+  ScrollView,
+  Slider,
+  Switch,
+  TextInput,
+  ToolbarAndroid,
+  ViewPagerAndroid,
+  DrawerLayoutAndroid,
+  FlatList,
+  TouchableWithoutFeedback,
+} from 'react-native';
+import gestureHandlerRootHOC from './gestureHandlerRootHOC';
+import Directions from './Directions';
+import State from './State';
+
+// Factory for stub Handler components
+function createStubHandler(name) {
+  return class NativeViewGestureHandler extends React.Component {
+    static displayName = name;
+
+    container = React.createRef();
+
+    setNativeProps() {
+      // Since this is a stub we do not need to pass on native props.
+      // However, we need to implement it here to avoid null calls.
+      // If for any reason we need to pass on native props we can fallback to this:
+      // this.container.current.setNativeProps(...args)
+    }
+
+    render() {
+      const { children, ...rest } = this.props;
+
+      // We don't want to create another layer, so instead we clone it only but keep the reference
+      const child = React.Children.only(children);
+      return React.cloneElement(child, {
```

Comment:
https://reactjs.org/docs/forwarding-refs.html

Is it actually needed to clone component for obtaining ref?

Reply:
This was something left over from the initial version, sorry.

Reply:
how about:
```
render() {
	return this.children
}
```